Day1/applesAndOranges.js

```diff
@@ -0,0 +1,27 @@
+function countApplesAndOranges(s, t, a, b, apples, oranges) {
+  //for apples
+  let applesLand = [];
```

Comment:
I created an array where I stored the thrown oranges, and the same for the apples.